portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramDetail.vue

```diff
@@ -152,7 +152,12 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
           @ok="deleteRule" />
       </div>
     </div>
-    <engagement-center-program-deleted v-else @back-to-list="backToProgramList" />
+    <engagement-center-result-not-found
+      v-else
+      @back-to-list="backToProgramList"
+      :message-title="$t('programs.details.programDeleted')"
+      :button-text="$t('programs.details.programDeleted.explore') "
+      :button-url="programsUrl" />
```

Comment:
Add event at the end

```suggestion
      :message-title="$t('programs.details.programDeleted')"
      :button-text="$t('programs.details.programDeleted.explore') "
      :button-url="programsUrl"
      @back-to-list="backToProgramList" />
```